a3700/mv_ddr_tim.h

```diff
@@ -359,7 +379,8 @@ static char ddr4_1cs_tim_post[] = {
 	"DELAY: 0x0000000A\n"
 	"\n"
 	";Step9: DDRPHY Driver/Receiver & DQS internal Pullup/Pulldown settings\n"
-	"WRITE: 0xC0001004 0xD0133449\n"
+	";WRITE: 0xC0001004 0xD0133449\n"
```

Comment:
You can remove this `;WRITE` line, it does nothing. Lines which start with semicolon are ignored.

Reply:
I thought it might be interesting for someone that the change is reflected (as Globalscaletech left it in like that). But on git, there's a history of all changes anyhow, so yes, I could as well just delete it. :-)

Reply:
Yes, it is in git, so it does not make sense to have code commented in source files.

Reply:
@heaterC: Could you also squash "removed commentary line" fix commit into "change DDR4-1cs for EspressoBin" commit? It does not have to be in two commits.